src/components/Dashboard/RightClick.vue

```diff
@@ -207,11 +211,22 @@ const menuData = computed<RightClickMenuEntryType[]>(() => [
     disabled: maindataStore.tags.length === 0,
     disabledText: t('dashboard.right_click.tags.disabled_title'),
     disabledIcon: 'mdi-tag-off',
-    children: maindataStore.tags.map(tag => ({
-      text: tag,
-      icon: hasTag(tag) ? 'mdi-checkbox-marked' : 'mdi-checkbox-blank-outline',
-      action: async () => await toggleTag(tag)
-    }))
+    children: [
+      ...(torrent.value?.tags.length
+        ? [
+            {
+              text: t('dashboard.right_click.tags.removeall'),
```

Comment:
Rename locale key to match format

```suggestion
              text: t('dashboard.right_click.tags.remove_all'),
```